docker-compose.yml

```diff
@@ -46,11 +42,7 @@ services:
     build:
       context: .
       target: sylius_nodejs
-      cache_from:
-        - quay.io/sylius/php:latest
-        - quay.io/sylius/nodejs:latest
-        - quay.io/sylius/nginx:latest
-    image: quay.io/sylius/nodejs:latest
+    image: nodejs:latest
```

Comment:
`nodejs` not is available image, the correct is `node`

Reply:
I have changed any nodejs to node in files.